platformio.ini

```diff
@@ -100,3 +100,11 @@ upload_speed = 921600
 ;  ${env.build_flags}
 ;  -DESP32C3
 ;board = lolin_c3_mini
+
+; If you want to use a ESP32C6, you can use this (experimental)
+;[env:esp32c6]
+;platform =  https://github.com/tasmota/platform-espressif32/releases/download/2024.04.14/platform-espressif32.zip
```

Comment:
Pls update as on main now. I belive that should work now.

Reply:
https://docs.platformio.org/en/latest/boards/espressif32/esp32-c6-devkitc-1.html#frameworks <- unfortunately not :( Arduino Framework is still not supported by the official platformio libs on the C6